ProceduralForest/Structures/World.cpp

```diff
@@ -66,10 +67,17 @@ void World::ProcedurallyGenerateWorld() {
     world_entities.push_back(terrain);
     for(int i = -30; i < 31; i++){
         for(int j = 0; j > -61; j--){
-            if ((rand() > 0.9975 * RAND_MAX)) {
+            if ((RandNumGen(0, RAND_MAX) > 0.9975 * RAND_MAX)) {
                 AddEntities(TreeGenerator::generateTree(vec3{i, 0, j}));
             }
         }
     }
     std::cout << 'h' << std::endl;
-}
\ No newline at end of file
+}
+
+int World::RandNumGen(int min, int max) {
```

Comment:
std::mt19937 is uint, you need to add a check and ensure that max is greater than min and that min is greater or equal to 0. Otherwise we could have unpredictable results